### PR TITLE
services/netauth: document initial setup.

### PR DESCRIPTION
**Pull request by ericonr**

### CODE REVIEW

File: docs/src/services/netauth.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # NetAuth
 
 NetAuth provides all the authentication and authorization information
-to systems within Void's managed fleet.  NetAuth is an open source
+to systems within Void's managed fleet. NetAuth is an open source
```
Comment (the-maldridge) [inline docs/src/services/netauth.md:4]: Style in this repo is 2 spaces after periods.

File: docs/src/services/netauth.md [hunk 2]
```diff
@@ -82,3 +82,21 @@ the key comment should be sufficient to remove it if it is unique.
...
+
+An initial config file for NetAuth can be obtained from the [void-infrastructure
+repository](https://github.com/void-linux/void-infrastructure/blob/master/ansible/roles/netauth-config/files/config.toml).
+It can be stored in `~/.netauth/config.toml`, for example, and should be
+modified so that the `tls.certificate` key points to a file containing the
+certificate for the <netauth.voidlinux.org> domain. The certificate can be
+obtained as shown below:
+
+```shell
+$ openssl s_client -showcerts -connect netauth.voidlinux.org:1729 </dev/null | openssl x509 -outform pem
```
Comment (the-maldridge) [inline docs/src/services/netauth.md:98]: Please also show the cfssl/jq version, it was generally nicer in my opinion.

**Review by the-maldridge — CHANGES_REQUESTED** [review]

**Comment by ericonr** [comment]
Done

**Comment by ericonr** [comment]
Ping @the-maldridge

**Pull request merged**